shared/slave_protocol.h

```diff
@@ -23,6 +23,7 @@
         SlaveCommand_RequestKeyStates,
         SlaveCommand_SetTestLed,
         SlaveCommand_SetLedPwmBrightness,
+        SlaveCommand_ResetTrackpoint,
```

Comment:
I feel uneasy about polluting `slave_command_t` with module-specific commands. We may also run out of command ids sooner, eventually this way. I first thought about adding a generic module reset command, but I'm unsure whether we'd use it for other modules later. I think having `SlaveCommand_ModuleSpecificCommand` is the best way forward, and using the second byte with its own enum per module as the actual module-specific command id.